### PR TITLE
Casptone

### PR DESCRIPTION
**Pull request by Meri-MG**
This is the capstone project PR.
In this pull request, several changes were made, such as: 
- All necessary files setup
- Added all necessary content to index.html
- Added all necessary content to about.html
- Added mobile CSS styles
- Added desktop CSS styles
- Created dynamic HTML page in JavaScript.
- Created dynamic Mobile Menu in JavaScrip.
- Updated Readme file.
- Fixed Linter Errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,73 @@
...
+# Built with
+- HTML5
+- CSS3
+- JavaScript
+- Git and GitHub
+- VScode
+
+# Live Demo
+> Project live demo [link](https://meri-mg.github.io/Capstone-Module-01/);
+> video presentation [link](https://www.loom.com/share/8f1add5499b944daa0f93166f62f895f);
```
Comment (SajjadAhmad14) [inline README.md]: Video presentation does not have clear audio. Please record a video that has clear audio.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        <h3>Partner</h3>
+        <hr />
+        <ul class="partners">
+            <li class="partner"><a href="" title="partners"><img src="images/trip.png" alt=""></a></li>
+            <li class="partner"><a href="" title="partners"><img src="images/Adidas.png" alt=""></a></li>
+            <li class="partner"><a href="" title="partners"><img src="images/Airbnb.png" alt=""></a></li>
+        </ul>
+    </section>
+</div>
+    <footer class="container footer about-footer">
```
Comment (SajjadAhmad14) [inline index.html:117]: Footer text on the desktop should be on the right side.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Meri Gogichashvili, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
You have followed the design almost on all pages. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by dasileker — APPROVED** [review]
### **Hi @Meri-MG,** 

### **Congratulations! 🎉 Project Approved ✅💯**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 


Approved
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**